defs/bitcoin/pivx.json

```diff
@@ -0,0 +1,40 @@
+{
+  "coin_name": "PIVX",
+  "coin_shortcut": "PIVX",
+  "coin_label": "PIVX",
+  "website": "https://pivx.org",
+  "github": "https://github.com/PIVX-Project/PIVX",
+  "maintainer": "Random Zebra <random.zebra@protonmail.com>",
+  "curve_name": "secp256k1",
+  "address_type": 30,
+  "address_type_p2sh": 13,
+  "maxfee_kb": 100000,
+  "minfee_kb": 100,
+  "signed_message_header": "DarkNet Signed Message:\n",

```

Comment:
Is this intentional?

Reply:
Yes (https://github.com/PIVX-Project/PIVX/blob/ed0dd2a20a87e9de56e9ff33387a8e68043fe840/src/main.cpp#L113).

DarkNet (DNET) launched in February 1st, 2016 and rebranded as 'PIVX' one year later (https://pivx.org/pivx-rebranding-complete-and-new-wallet-available/)